### DISCUSSION TITLE
What database is Photoprism using in this script?

### DISCUSSION THREAD

**Discussion by CodingEmil** [tteck/Proxmox#902] [Q&A] [answered]
Hello guys, pretty simple question, but I could find an answer myself...
Can anyone tell me if this script is setting up Photoprism with MySQL or MariaDB?
Thank you!

**Comment by tteck** [comment, marked as answer]
sqlite

https://docs.photoprism.app/getting-started/config-options/#database-connection

**Reply by CodingEmil** [reply]
have you thought about changing to go along with mariaDB, the Photoprism creators highly recommend it.

**Reply by tteck** [reply]
No, the end user can change to another supported DB if they wish.

**Reply by besforttt** [reply]
Can you elaborate how is that done?

All the online documentation I can find, it only references docker install method!
Being able to switch to MariaDB from SQLite3 would be awesome for me, it's the third time I keep re-indexing images (around 70k) only to have something go wrong with sqlite3.
The resources are not an issue as I've allocated 16 vCores, 512GB NVMe SSD and 32GB of RAM to it!

If a migration is not possible, I am willing to re-install from scratch just as long as I'd be able to use MariaDB as a database server.

Thanks,
besforttt

**Reply by bas89** [reply]
Entere the console of the LXC. `cd /opt/photoprism/config` and there you edit the config via `nano .env`. Here, you can set the server to mysql and point it to your, i.e., mariadb server. Of course, you have to set it up first, but the documentation at photoprism helps you with that.

This file won't be overwritten by updating, right @tteck ?
